### PR TITLE
fix(quick-install): disable minio versioning

### PR DESCRIPTION
**Pull request by lwander**

### CODE REVIEW

**Comment by lwander** [comment]
@sbwsg feel free to merge this after your change & i'll fix the merge conflicts

**Review by ghost — APPROVED** [review]
Nice, lgtm

**Comment by ghost** [comment]
aw crap.  i meant to leave a comment but i accidentally added a commit message.  here was the comment i meant to leave:

i'm just doing some last minute sanity checks on my changes so i'll do the merge conflicts as well

**Pull request merged**
